### PR TITLE
SUS-458 | CNW: NS_PROJECT pages are missing on new wikis

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-458

Use a proper content language (`$wgContLang`) for a given starter to be able to generate localized namespace names.

Set `$wgMetaNamespace` to properly generate `NS_PROJECT` namespace names:

```xml
    <namespaces>
      <namespace key="-2" case="first-letter">Media</namespace>
      <namespace key="-1" case="first-letter">Speciale</namespace>
      <namespace key="0" case="first-letter" />
      <namespace key="1" case="first-letter">Discussione</namespace>
      <namespace key="2" case="first-letter">Utente</namespace>
      <namespace key="3" case="first-letter">Discussioni utente</namespace>
```

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [dumpStarters.php#L16](https://github.com/Wikia/app/blob/cd586608d97cd8774cf3d4466e967d17fe67422c/extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php#L16): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+ * s3cmd -c /etc/s3cmd/sjc_prod.cfg ls s3://starter/dumps/
  *
  * @see PLATFORM-1305
  *
  * @author Macbre
  * @ingroup Maintenance
  */
 
-putenv('SERVER_ID=80433'); // run in the context of www.wikia.com (where CreateWiki is enabled)
+putenv( 'SERVER_ID=80433' ); // run in the context of www.wikia.com (where CreateWiki is enabled)
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php [hunk 2]
```diff
@@ -1,17 +1,19 @@
 <?php
 
 /**
- * Script that prepares XML dumps with the latest revisions of starter wikis
+ * Script that prepares XML and SQL dumps with the latest revisions and *links tables rows of starter wikis
```
Comment (ghost) [inline extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php:4]: `and *links…` - it seems a newline is missing

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php (hunk 1 above)
Comment (ghost) [inline extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php:16]: Does it make sense to use a named constant here and across all codebase?

**Review by ghost — CHANGES_REQUESTED** [review]

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php (hunk 2 above)
Comment (macbre) [reply extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php:4]: Nope, `*links` refers to `categorylinks`, `externallinks`, ... tables - see `$mTablesToDump` class member below.

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php:16]: It's actually too early to use constant defined in another file

**Review by sqreek — APPROVED** [review]

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php:16]: INFO Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**